### DISCUSSION TITLE
Scripts with import statement from other file

### DISCUSSION THREAD

**Discussion by andershagman** [netbox-community/netbox#14108] [Help Wanted!] [answered]
Running version 3.6.4 I have a problem moving my scripts from 3.3.6.
It seems that you no longer can just place your files in the script directory.
You have to use the GUI.

My problem is that my script isn´t one file.
I have a directory under the script directory with common stuff and import that into my script.
Problem is that I can´t upload multiple files in the GUI.

How do I handle this?

**Comment by kkthxbye-code** [comment]
There's no great way to handle this in 3.5+. We moved to a plugin for our custom scripts instead:

https://github.com/kkthxbye-code/netbox-script-manager

If that's not an option, you might be able to place your dependencies in the backing script directory manually and then upload only the main script file. It's a mess to keep in sync though with some files originating from netbox and some files manually kept up to date.

**Reply by andershagman** [reply]
A plugin is a big workaround.

Unfortunate copy the library directory does not work. The script file can not find the files.

**Comment by andershagman** [comment]
How do we handle passwords, tokens etc?
How can we handle situations if we use a config.py file?

**Reply by andershagman** [reply, marked as answer]
It works!

import sys
sys.path.append('/srv/netbox/shared/scripts')
